### DISCUSSION TITLE
Can another Organization owner kick me out?

### DISCUSSION THREAD

**Discussion by adamhelton** [community/community#22964] [New to GitHub]
<p>Hi,</p>
<p>I’m currently the sole owner of an organization for an internship at my school. I’ve got a few members, but some people from another department have stated they want to be put as an owner for the organization.</p>
<p>My question is if I make them an owner, can they then bump my priveledges down from being an owner to just a member?</p>
<p>Also, could I do the same to them?</p>
<p>I don’t know how permanent the owner role is I guess</p>

**Comment by seveas** [comment, marked as answer]
<p>Yes, an owner has full access to the organization and can make the changes you describe.</p>
